libs/orders/src/lib/order-hooks/use-order-edit.tsx

```diff
@@ -50,11 +50,14 @@ export const useOrderEdit = () => {
           orderAmendment: {
             orderId: order.id,
             marketId: order.market.id,
+            // @ts-ignore fix me please!
             price: {
               value: removeDecimal(order.price, order.market?.decimalPlaces),
             },
             timeInForce: VegaWalletOrderTimeInForce[order.timeInForce],
+            // @ts-ignore fix me please!
```

Comment:
yarn install - so that it pulls latest version  of `@vegaprotocol/vegawallet-service-api-client` should fix this. anyway you might update this in the next PR